build-and-deploy-cloud-functions.sh

```diff
@@ -0,0 +1,45 @@
+PROJECT_ID=$1
+STUDY_MANAGER_SCHEMA=$2
+STUDY_SERVER_SCHEMA=$3
+
+echo "Will deploy to ${PROJECT_ID} using schemas ${STUDY_MANAGER_SCHEMA} and ${STUDY_SERVER_SCHEMA}"
+
+ mvn -Pcloud-function -DskipTests clean install package
+
+
+echo "Deploying kit dispatcher to ${PROJECT_ID}"
```

Comment:
can we make that deploy script generic or do we always want to re-deploy all cfs?

Reply:
I'm in favor of always re-deploying so that we have something nice and simple: whenever one CF is updated, they're all updated.